test/basic.js

```diff
@@ -561,6 +608,7 @@ util.testSinglePlatform('tmpdir test', createDisableTmpdirUsingTest)
 util.testSinglePlatform('ignore out dir test', createIgnoreOutDirTest, 'ignoredOutDir')
 util.testSinglePlatform('ignore out dir test: unnormalized path', createIgnoreOutDirTest, './ignoredOutDir')
 util.testSinglePlatform('ignore out dir test: unnormalized path', createIgnoreImplicitOutDirTest)
+util.testSinglePlatform('deref symlink test', createDisableSymlinkDereferencingTest)
```

Comment:
My suspicion is that this test will need to be disabled when running on Windows, but AppVeyor CI will tell us for sure.